### DISCUSSION TITLE
self-close `img` tag in `hastscript`?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [syntax-tree/unist#89] [Q&A] [answered]
i did `h('img.frontcover')` & it generated `<img class="frontcover">`

curious, why it isn't self-closed? `input` works fine as shown in the readme but not `img`

any reasons? or bug?

**Comment by wooorm** [comment, marked as answer]
hastscript doesnt generate a string of html markup. hast-util-to-html does that. It has an option for this. But you don't need it. This is html. That slash doesn't so anything. Like semicolons in most places in javascript, or single vs double quotes.
